pom.xml

```diff
@@ -90,11 +90,11 @@
                module. Use the contents of that for OUR version of startup.properties, but don't
                forget to replace the felix config admin bundle with OUR platform-osgi-configadmin
                bundle if you copy-pasted. -->
-        <karaf.version>4.1.2</karaf.version>
+        <karaf.version>4.1.4</karaf.version>
```

Comment:
The karaf version should probably be updated here:

- https://github.com/codice/ddf/blob/1d77016539f4862a106793ea5f4c72a3bf0e353d/dependency-check-maven-config.xml#L18
- https://github.com/codice/ddf/blob/2468377a6782552e1f3ea0115ea48793f1809a77/distribution/ddf-common/src/main/resources-filtered/etc/custom.properties#L56
- https://github.com/codice/ddf/blob/cce276f3e2528025ebc201fc12b638c0b2adcc94/distribution/test/itests/test-itests-common/src/main/java/org/codice/ddf/itests/common/AbstractIntegrationTest.java#L113

Looks like none of the enterprise features changed, so the karaf version should probably be updated here too:

- https://github.com/codice/ddf/blob/02926fc184a9b4bdbfc882a3767e236819191e66/platform/platform-app/src/main/resources/features.xml#L266
- https://github.com/codice/ddf/blob/250eafa28086f6318bb68293f1a4e7b518e4d407/platform/platform-app/pom.xml#L396

Reply:
Does https://github.com/codice/ddf/blob/cce276f3e2528025ebc201fc12b638c0b2adcc94/distribution/test/itests/test-itests-common/src/main/java/org/codice/ddf/itests/common/AbstractIntegrationTest.java#L506-L518 need to be updated to match https://github.com/apache/karaf/blob/karaf-4.1.4/assemblies/features/base/src/main/resources/resources/etc/org.ops4j.pax.url.mvn.cfg#L114-L117?

Reply:
Should the `aries.blueprint.core.version` here be updated from 1.8.2 to 1.8.3
https://github.com/codice/ddf/blob/b57f28fff72814d0c4383d6f48b95ead038f4a58/distribution/ddf-common/src/main/resources/common-bin.xml#L61?

Reply:
All good finds. Made all updates.